src/main/java/ee/carlrobert/codegpt/action/ActionsUtil.java

```diff
@@ -43,7 +51,23 @@ public static void refreshActions(Map<String, String> tableData) {
       tableData.forEach((action, prompt) -> group.add(new BaseAction(action) {
         @Override
         protected void actionPerformed(Project project, Editor editor, String selectedText) {
-          sendMessage(project, prompt.replace("{{selectedCode}}", format("\n\n%s", selectedText)));
+          Pattern pattern = Pattern.compile("def\\s+(self\\.)?(\\w+)");
+
+          // Create a matcher object for the given string
+          Matcher matcher = pattern.matcher(selectedText);
+          String functionName = "";
+
+          // Check if the pattern matches and extract the function name
+          if (matcher.find()) {
+             functionName = matcher.group(2);
```

Comment:
**Suggestion:** The 'functionName' variable is initialized as an empty string and then possibly reassigned in a conditional statement. This can be simplified by initializing 'functionName' within the if statement.
```suggestion
          Matcher matcher = pattern.matcher(selectedText);

          if (matcher.find()) {
             String functionName = matcher.group(2);
          }
```